CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
+# 3.0.0-rc.6 (2019/12/04)
+
+- Fixed the keyboard navigation so that hitting enter while editing a cell saves it and starts editing the next in the column. [#20](https://github.com/blackbaud/skyux-ag-grid/pull/20)
```

Comment:
What do you think of tweaking this to:

`Fixed the keyboard navigation so that users can press Enter to save a cell and switch focus to the next column.`